template_data/v2/invalid.html

```diff
@@ -83,7 +83,7 @@ <h3 class="title">Please Note</h3>
         <div class="supplemental__about">
           <h2 class="title">About edX</h2>
           <div class="copy">
-            <p>EdX offers interactive online classes and MOOCs from the world’s best universities, including MIT, Harvard, Berkeley, University of Texas, and many others.  EdX is a non-profit online initiative created by founding partners Harvard and MIT.</p>
+            <p>EdX offers interactive online classes and MOOCs from the world’s best universities, including MIT, Harvard, Berkeley, University of Texas, and many others.  EdX is an online initiative created by founding partners Harvard and MIT.</p>
```

Comment:
I think we should align this sentence with the sentence in `edx-themes` or wise versa. 
https://github.com/edx/edx-platform/blob/master/themes/edx.org/lms/templates/certificates/_about-edx.html

`edX is an online initiative` => `edX is a mission driven initiative`